db/user.js

```diff
@@ -43,6 +43,16 @@ class UserDbController {
     })
   }
 
+  async linkToRegistry ({ userId, registry, data }) {
```

Comment:
what is this used for? what is registry data here?

Reply:
ohhhh users have registry usernames. maybe just a comment would help

Reply:
adding comment